site/content/docs/manifest/lb-web-service.en.md

```diff
@@ -27,6 +27,12 @@ List of all available properties for a `'Load Balanced Web Service'` manifest. T
         dockerfile: ./frontend/Dockerfile
         context: ./frontend
       port: 80
+      healthcheck:
```

Comment:
nit: What do you think of removing this from the sample LB Web Service manifest? I don't want to confuse folks with two types of healtchecks, instead the HTTP healthcheck will be the most common thing they care about for web services?

Reply:
good point